EIPS/eip-8007.md

```diff
@@ -0,0 +1,52 @@
+---
+eip: 8007
+title: Glamsterdam Gas Repricings
+description: Directory of EIPs introducing changes to the gas pricing model for the Glamsterdam fork
+author: Maria Silva (@misilva73), Ansgar Dietrichs (@adietrichs)
+discussions-to: https://ethereum-magicians.org/t/eip-8007-glamsterdam-gas-repricings-meta-eip/25206
+status: Draft
+type: Meta
+created: 2025-08-21
+---
+
+## Abstract
+
+This Meta EIP documents all the proposals for Glamsterdam related to the gas repricing effort. The goal of this effort is to harmonize gas costs across the EVM, thereby reducing the impact of specific bottlenecks on scaling. Proposals include changes to the cost of single EVM operations, as well as bigger changes to the gas model. This Meta EIP is purely informational and does not aim to have an active role in the governance process for the Glamsterdam fork. Instead, it serves as a directory for all repricing-related proposals, helping to organize the work and keeping the community informed about the status of each EIP.
+
+## Motivation
+
+The main objective of the Glamsterdam fork is to improve L1 scalability. A crucial aspect of this initiative is to create a better alignment between gas costs and actual resource usage. Currently, the gas model often misprices operations, resulting in inefficiencies and unintended incentives. For instance, within the pure compute operations, there is a high variance in execution time per gas unit, which indicates that a single unit of computation is not priced equally across the various opcodes.
+
+By standardizing gas costs across EVM operations and other resources, we can reduce bottlenecks and enhance the utilization of EVM resources, which will subsequently enable further scalability. The EIPs listed below constitute a significant first step in that direction. We expect that further iteration will be necessary in future hardforks.
+
+## Specification
+
+This Meta EIP includes two types of EIPs, namely, core and possible additions. This list will continue to be updated as more gas repricing EIPs are proposed.
+
+### Core EIPs
+
+| EIP                       | Description                                                                                | Type               | Status                 |
+| ------------------------- | ------------------------------------------------------------------------------------------ | ------------------ | ---------------------- |
+| [EIP-7904](./eip-7904.md) | Gas Cost Repricing to reflect computational complexity and transaction throughput increase | Compute            | Pending                |
```

Comment:
can you put all the mentioned EIPs in the required field of the EIP, this will help bots evaluate if this EIP is in ready statue (and that its requirements are all merged and in ahead status)

any eips that you don't require when going to review, you may remove at that point from the required list